ios/FluentUI/Card Nudge/CardNudge.swift

```diff
@@ -80,10 +81,10 @@ public struct CardNudge: View, ConfigurableTokenizedControl {
     var textContainer: some View {
         VStack(alignment: .leading, spacing: tokens.interTextVerticalPadding) {
             Text(state.title)
-                .font(.subheadline)
-                .fontWeight(.medium)
+                .font(.fluent(tokens.aliasTokens.typography[.body2Strong], shouldScale: false))
```

Comment:
Can we move these tokens to a control-level token in `CardNudgeTokens`? I'd rather we avoid directly referencing global or alias tokens directly inside components.

Reply:
Good call! I was following the previous implementation, but I agree, design tokens should be inside the tokens class. :)